internal/resolver/dns/dns_resolver_test.go

```diff
@@ -1175,7 +1175,7 @@ func TestResolveFunc(t *testing.T) {
 		{"[2001:db8:a0b:12f0::1]:21", nil},
 		{":80", nil},
 		{"127.0.0...1:12345", nil},
-		{"[fe80::1%lo0]:80", nil},
+		// {"[fe80::1%lo0]:80", nil}, // TODO(easwars): url.Parse raises: "invalid URL escape "%lo" [recovered]"
```

Comment:
We expect all of dial targets to parse as URL's as defined in RFC 3986. The `%` in this IPv6 address needs to be escaped. So, if we replace this target as `[fe80::1%25lo0]:80`, it should parse.

Reply:
Done. Thanks for your help.